Makefile

```diff
@@ -0,0 +1,17 @@
+all: build lint
```

Comment:
nit: It seems like lint should be run before build to me. The outcome wont be any different but seems more logical that way